tests/uint128_t.cpp

```diff
@@ -148,49 +115,7 @@ BOOST_AUTO_TEST_CASE(test_basic)
     BOOST_CHECK_EQUAL( id6.low(), 0xE9800998ECF8427Eull );
 }
 
-BOOST_AUTO_TEST_CASE(test_perf_test)
```

Comment:
This test was useful because it exhibited errors in the random number generator usage. Why not keep it in the C++11 codepath?